dwave/preprocessing/presolve/cypresolve.pyi

```diff
@@ -24,5 +32,6 @@ class cyPresolver:
     def clear_model(self) -> None: ...
     def copy_model(self) -> dimod.ConstrainedQuadraticModel: ...
     def detach_model(self) -> dimod.ConstrainedQuadraticModel: ...
+    def feasibility(self) -> Feasibility: ...
```

Comment:
How can presolve get a feasible solution to tell the model is feasible? Is it only for trivial cases?

Reply:
It will usually be `Unknown` or `Infeasible`, but there are some cases (like if all variables are fixed) that it can mark `Feasible`.